turbo/execution/eth1/forkchoice.go

```diff
@@ -276,15 +278,6 @@ func (e *EthereumExecutionModule) updateForkChoice(ctx context.Context, original
 			return
 		}
 
-		// If we don't have it, too bad
```

Comment:
why remove this check and how that related to metrics?

Reply:
I found that the same check happens at line 241
and we are already in if block ```if fcuHeader.Number.Uint64() > 0 ``` which would cause panic if fcuHeader is nil

Reply:
there are different error returned